docs/contributing/community.md

```diff
@@ -32,7 +32,7 @@ The Gatsby community welcomes contributions. Please refer to the guides below on
 - [How to Open a Pull Request](/contributing/how-to-open-a-pull-request/)
 - [Make feature requests with an RFC](/blog/2018-04-06-introducing-gatsby-rfc-process/)
 - [Gatsby Style Guide](/contributing/gatsby-style-guide/)
-- [Gatsby Translations](/contributing/gatsby-docs-translation-guide/)
+- [Gatsby Translations - currently on hold](/contributing/translation/)
```

Comment:
I updated the link - it was redirecting to it.

Reply:
Good catch! Thanks :)